core/src/main/java/greencity/webcontroller/ManagementRatingStatisticsController.java

```diff
@@ -31,15 +36,18 @@
     private RatingStatisticsService ratingStatisticsService;
     private RatingExcelExporter ratingExcelExporter;
     private final DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
+    private RatingStatisticsSpecification spec;
 
     /**
      * Constructor.
      */
     @Autowired
     public ManagementRatingStatisticsController(RatingStatisticsService ratingStatisticsService,
-                                                RatingExcelExporter ratingExcelExporter) {
+                                                RatingExcelExporter ratingExcelExporter,
+                                                RatingStatisticsSpecification spec) {
         this.ratingStatisticsService = ratingStatisticsService;
         this.ratingExcelExporter = ratingExcelExporter;
+        this.spec = spec;
```

Comment:
`RatingStatisticsSpecification` must be created at Service level. A new one per each request.